attic/readme.md

```diff
@@ -7,3 +7,4 @@ Code, documentation, experiments we want to retain for reference, but that we're
     * note: `langevin_coefficients` dependency has since changed -- some scripts rely on a version of `langevin_coefficients` prior to PR #459
 * `training/` -- classes and functions for an earlier training workflow
     * note: somewhat tailored to use TI estimates, constant volume simulations, GRPC, ...
+* `scripts/` -- script for approximating AM1BCC with atom types rather than bond types
```

Comment:
@proteneer : confirm this description?

Reply:
nit: maybe specify the filename here since the description only applies to the one script (otherwise might get confusing if more scripts are added and we forget to update this)

Reply:
Good call -- addressed in https://github.com/proteneer/timemachine/pull/592/commits/8e32e55fdfd9233fa77aa5f9e6177585073d46fd !